docs/File-Formats.md

```diff
@@ -718,67 +772,9 @@ You can learn more about configuring these annotations in the [portal.properties
-
-An example cBioPortal mutation data file with the following **additional** columns:
-```
-ASCN.total_copy_number    ASCN.clonal     MUTATION.name    MUTATION.type
-```
-imported with the following `namespaces` field in the metafile:
-```
-namespaces: ascn
-```
-will import the `ASCN.total_copy_number` and `ASCN.clonal` column into the database. `MUTATION.name` and `MUTATION.type` will be ignored because `mutation` is not specified in the `namespaces` field. 
-
-## Representation of namespace columns by mutation API endpoints
-
-Columns added through namespaces will be returned by mutation API endpoints. Namespace data will be available in
-the `namespaceColumn`
-of respective JSON representations of mutation records. The `namespaceColumns` property will be a JSON object where
-namespace data is keyed by name of the namespace in lowercase. For instance, when namespace `ZYGOSITY` is defined in the
-meta file and the data file has column `ZYGOSITY.status` with value `Homozygous` for a mutation row, the API will return
-the following JSON record for this mutation (only relevant fields are shown):
-
-```
-{
-    "namespaceColumns": {
-        "ZYGOSITY": {
-            "status": "Homozygous"
-        }
-    },
-}
-```
-
-Note: ASCN namespace data is not exported via the `namespaceColumns` field.
-
-## Representation of namespace columns in the cBioPortal frontend
-
-Namespace columns will be added as columns to mutation tables in Patient View and Results View. The case of the
-namespace in the column header will be as specified in the mutations meta file and the column name will be capitalized.
-For instance, this metafile entry:
-
-```shell
-namespaces: Zygosity
-```
-
-and this column header:
-
-```shell
-ZYGOSITY.status
-```
-
-will show in the mutation table with column name:
-
-```shell
-Zygosity Status
-```
-
-Note: namespace columns are recognized by a case-insensitive match of the namespace reported in the mutations meta file
-and the first word in the column header.
+Additional mutation annotation columns can be added to the cBioPortal mutation data file. In this way, the portal will parse and store your own MAF fields in the database. For example, mutation data that you find on cBioPortal.org comes from MAF files that have been further enriched with information from [mutationassessor.org](https://mutationassessor.org/), which leads to a "Mutation Assessor" column in the [mutation table](https://www.cbioportal.org/index.do?cancer_study_list=acc_tcga&cancer_study_id=acc_tcga&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=acc_tcga_mutations&Z_SCORE_THRESHOLD=2.0&RPPA_SCORE_THRESHOLD=2.0&data_priority=0&case_set_id=acc_tcga_sequenced&case_ids=&patient_case_select=sample&gene_set_choice=user-defined-list&gene_list=ZFPM1&clinical_param_selection=null&tab_index=tab_visualize&Action=Submit).
```

Comment:
Why do we reference an `index.do` link here instead of the results page? I thought that stuff was all deprecated.